### PR TITLE
add qgb orchestrator address logic

### PR DESCRIPTION
**Pull request by SweeXordious**
Adds QGB orchestrator address logic.

Contributes to: https://github.com/celestiaorg/celestia-app/issues/197

### CODE REVIEW

**Review by evan-forbes — APPROVED** [review]
LGTM :+1:

File: x/qgb/keeper/keeper_delegate_key.go [hunk 1]
```diff
@@ -2,13 +2,205 @@ package keeper
...
+				CommissionRates: stakingtypes.CommissionRates{
+					Rate:          sdk.Dec{},
+					MaxRate:       sdk.Dec{},
+					MaxChangeRate: sdk.Dec{},
+				},
+				UpdateTime: time.Time{},
+			},
+			MinSelfDelegation: sdk.Int{},
+		}, false
+	}
```
[the remark this answers is not in the document]
Comment (SweeXordious) [reply x/qgb/keeper/keeper_delegate_key.go:132]: Nice catch :+1:. I checked, it is introduced after adding the [`exhaustivestruct`](https://github.com/mbilski/exhaustivestruct) linter to the project in here: https://github.com/Gravity-Bridge/Gravity-Bridge/commit/df00ccaf15d3c0be7ce743ccd65986a9e6f0fd58
I guess, we can change it in our case to: 
```suggestion
		return stakingtypes.Validator{}, false
```
What do you think ?

[the remark this answers is not in the document]
Comment (SweeXordious) [reply x/qgb/keeper/keeper_delegate_key.go:132]: In fact, we can even go for what I suggest in the next comment.

File: x/qgb/keeper/keeper_delegate_key.go [hunk 2]
```diff
@@ -2,13 +2,205 @@ package keeper
...
+		Commission: stakingtypes.Commission{
+			CommissionRates: stakingtypes.CommissionRates{
+				Rate:          sdk.Dec{},
+				MaxRate:       sdk.Dec{},
+				MaxChangeRate: sdk.Dec{},
+			},
+			UpdateTime: time.Time{},
+		},
+		MinSelfDelegation: sdk.Int{},
+	}, false
```
Comment (SweeXordious) [inline x/qgb/keeper/keeper_delegate_key.go:165]:
```suggestion
	// TODO look for the validator when the staking module is added
	return stakingtypes.Validator{}, false
```

File: x/qgb/keeper/keeper_delegate_key.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (evan-forbes) [reply x/qgb/keeper/keeper_delegate_key.go]: awesome, thanks for investigating and even fixing!

**Comment by SweeXordious** [comment]
@evan-forbes Let's merge if looks good.

**Pull request merged**